src/TLuaInterpreter.cpp

```diff
@@ -9953,18 +9935,15 @@ int TLuaInterpreter::echoPopup(lua_State* L)
     }
 
     Host& host = getHostFromLua(L);
-    QString txt = a2.c_str();
-    QString name = a1.c_str();
-    if (_commandList.size() != _hintList.size()) {
-        lua_pushstring(L, "Error: command list size and hint list size do not match cannot create popup");
-        lua_error(L);
-        return 1;
+    if (commandList.size() != hintList.size()) {
+        lua_pushfstring(L, "echoPopup: commands and hints list aren't the same size");
+        return lua_error(L);
     }
 
-    if (a1.empty()) {
-        host.mpConsole->echoLink(txt, _commandList, _hintList, customFormat);
+    if (window.isEmpty()) {
```

Comment:
Shouldn't this also handle `"main"` for consistency with (*some*) other functions.

Reply:
And always return `true` as a single argument to the caller when/if the other branch is upgraded to report that a valid window name was given...

Reply:
Updated and the method returns `void` right now, don't want to do deep surgery